src/main/java/edu/umich/its/cpm/Application.java

```diff
@@ -30,12 +17,28 @@
 	private static final Logger log = LoggerFactory
 			.getLogger(Application.class);
 
+	@Autowired
+	private MigrationInstanceService migrationInstanceService;
+
 	@Override
 	protected SpringApplicationBuilder configure(
 			SpringApplicationBuilder application) {
 		return application.sources(Application.class);
 	}
 
+	@PostConstruct
+	public void startMigrationProcessingThread() {
```

Comment:
There is no warning in UI. Mark as TODO for now. Meanwhile, there will be an error log for it.